src/app/api/health/route.ts

```diff
@@ -9,6 +9,8 @@ export async function GET(_: NextRequest) {
         version: packageJson.version,
         contracts_deployer: env.SBTC_CONTRACT_DEPLOYER,
         git_commit: env.GIT_COMMIT || "unknown",
+        withdrawal_fee_multiplier: env.WITHDRAWAL_FEE_MULTIPLIER,
+        withdrawal_max_fee: 180 * env.WITHDRAWAL_FEE_MULTIPLIER,
```

Comment:
Can we not have a magic number here and instead rely on a constant? Since this is also used elsewhere in the code.

Reply:
fair nuff